test/packages/bad_accept_ignore_above/changelog.yml

```diff
@@ -0,0 +1,26 @@
+- version: 1.0.0
```

Comment:
Test packages need to be referenced in `code/go/pkg/validator/validator_test.go` to be actually used in testing.